README.md

```diff
@@ -21,13 +21,29 @@ was originally developed by the [Dell CloudEdge Solutions Team](http://dell.com/
 ## Install
 
 To install this simple rubygem you can check your package manager if it is
-available for you, alternativly you can just install the ruby gem if you have a
+available for you, alternatively you can just install the rubygem if you have a
```

Comment:
one line above it's "rubygem" again :-) but I will change all of them to "ruby gem" if preferred